models/recruit/src/index.ts

```diff
@@ -502,6 +502,14 @@ export function createModel (builder: Builder): void {
     recruit.viewlet.VacancyApplicationsEmbeddeed
   )
 
```

Comment:
You need to remove all related to Archive functionality from Recruit, it 
<img width="621" alt="image" src="https://github.com/hcengineering/platform/assets/477235/17a2591a-fc10-4b03-a717-1460c63f28f8">